### PR TITLE
IDVA5-266 howIdentityDocumentsChecked

### PR DESCRIPTION
**Pull request by sardhani-ch**

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by swalke-ch — APPROVED** [review]

File: src/controllers/howIdentityDocumentsCheckedController.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import { Session } from "@companieshouse/node-session-handler";
+import { ClientData } from "model/ClientData";
+import { USER_DATA } from "../utils/constants";
+import { saveDataInSession } from "../utils/sessionHelper";
+
+export const get = async (req: Request, res: Response, next: NextFunction) => {
+    const lang = selectLang(req.query.lang);
+    const locales = getLocalesService();
+    const session: Session = req.session as any as Session;
+    const clientData: ClientData = session.getExtraData(USER_DATA) ? session.getExtraData(USER_DATA)! : {};
```
Comment (sampankumar) [inline src/controllers/howIdentityDocumentsCheckedController.ts:16]: Does having an empty value prevent nullpointer on line 22?

**Review by ttingle-ch — APPROVED** [review]
LGTM

**Pull request merged**